documentation/source/user_section/installation/mac.rst

```diff
@@ -23,7 +33,7 @@ You need to have ``gcc`` installed. Check to see if ``gcc`` is installed by open
     gcc --version
 
 
-If it isn't installed, we recommend installing `Homebrew <https://brew.sh/>`_ and then run:
+If it isn't installed, we recommend installing `Homebrew <https://brew.sh/>`__ and then run:
```

Comment:
Is there a reason that these external hyperlinks were changed to use anonymous targets? ([`_` vs `__`](https://stackoverflow.com/a/74258281))

e.g. Had you run into the error "Duplicate explicit target name"?

Reply:
I think anonymous targets are easier to think about, since they don't involve any global state (that is, the document-wide set of all targets, case-insensitive and whitespace-insensitive), so that's what I default to.

I think we should reserve named hyperlink references to cases where we actually want to refer to a URL multiple times, and in that case we should use the explicit format:
```rst
Here is some text mentioning a `carefully chosen link text`_.
And once again, this is a `Carefully Chosen Link Text`_ that is descriptive.

.. _Carefully chosen link text: https://www.example.com/
```

Reply:
That makes sense to me! I can open an issue to follow up on this for the rest of the documentation. :)